python/cudf/cudf/tests/test_reshape.py

```diff
@@ -596,6 +596,42 @@ def test_pivot_table_simple(data, aggfunc, fill_value):
     assert_eq(expected, actual, check_dtype=False)
 
 
+@pytest.mark.parametrize(
+    "data",
+    [
+        {
+            "A": ["one", "one", "two", "three"] * 6,
+            "B": ["A", "B", "C"] * 8,
+            "C": ["foo", "foo", "foo", "bar", "bar", "bar"] * 4,
+            "D": np.random.randn(24),
+            "E": np.random.randn(24),
+        }
+    ],
+)
+@pytest.mark.parametrize(
+    "aggfunc", ["mean", "count", {"D": "sum", "E": "count"}]
+)
+@pytest.mark.parametrize("fill_value", [0])
+def test_dataframe_pivot_table_simple(data, aggfunc, fill_value):
```

Comment:
What the difference between this test and `test_pivot_table_simple`? I can't immediately tell

Reply:
The previous test uses `cudf.pivot_table`, and the new test uses `DataFrame.pivot_table`. That is the only difference. I am open to alternatives here. I initially decided against a parametrization of the previous test but could be swayed to do whatever you think is right.

Reply:
Ah I see it now, thanks. Yeah I don't think it's worth parameterizing here.